charters/charter-2021.html

```diff
@@ -376,15 +370,12 @@ <h3 id="w3c-coordination">W3C Groups</h3>
             <dt><a href="https://www.w3.org/community/webtiming/">Multi-device Timing CG</a></dt>
             <dd>Support multi-device timing in media applications, such as multi-screen presentations</dd>
 
-            <dt><a href="https://www.w3.org/community/publishingcg/">Publishing CG</a> and <a href="https://www.w3.org/community/publishingbg/">Publishing BG</a></dt>
-            <dd>The Publishing Community Group functions as an incubator group to explore, test, and possibly specify new ideas that may be included, eventually, into later versions of EPUB 3.X. The Publishing Business Group fosters ongoing participation by members of the publishing industry and overall publishing ecosystem in the development of the Web for publishing, and serves as a conduit for feedback between the publishing ecosystem and W3C.</dd>
+            <dt><a href="https://www.w3.org/community/publishingbg/">Publishing BG</a></dt>
+            <dd>The focal point of the Publishing@W3C, which works on issues to make publications on the Web to be more capable, more beautiful, more accessible, and easier. The main expectation for the collaboration with MEIG is business and technical discussion on digital publishing with other media and entertainment, e.g., (1) immersive reading of digital publications with immersive web technologies, (2) <a href="https://www.w3.org/2011/audio/">audiobooks</a> in an interoperable manner, and (3) <a href="https://www.w3.org/community/sync-media-pub/">synchronized audio/video</a> with Web Publications.</dd>
```

Comment:
Just some editorial nits to expand "Publishing@W3C" and improve readability. I'll let @chrisn confirm/correct/complete with his native English speaker hat on :)

```suggestion
            <dd>The focal point of the publishing activity at W3C, which works on issues to make publications on the Web more capable, more beautiful, more accessible, and easier. The main expectation for the collaboration with MEIG is business and technical discussion on digital publishing linked to media and entertainment topics, e.g., (1) immersive reading of digital publications with immersive web technologies, (2) <a href="https://www.w3.org/2011/audio/">audiobooks</a> in an interoperable manner, and (3) <a href="https://www.w3.org/community/sync-media-pub/">synchronized audio/video</a> with Web Publications.</dd>
```

Reply:
Please note that the text was provided by not me but Daihei as the Publishing BG Chair :)
So we need to ask him to review the updated text again.

Reply:
These are editorial nits, I don't think they warrant another review loop.

We need to officialize the new charter as soon as possible because the IG is currently out of charter. I'm fine not incorporating my suggestions. As things stand, I just cannot fully parse the sentence as English, which seems wrong to me.

Reply:
I'm fine with Publishing@W3C, which is how the publishing activity labels itself: https://www.w3.org/publishing/. I suggest that "with other media and entertainment" needs some additional words to clarify its meaning, so "with other media and entertainment topics" seems OK given the examples that follow. Or is the intent more "with companies in the media and entertainment industry on topics such as [examples]"?

Reply:
Thanks for your comments, @tidoust and @chrisn !

So our proposal is:
[[
The focal point of the Publishing@W3C, which works on issues to make publications on the Web more capable, more beautiful, more accessible, and easier. The main expectation for the collaboration with MEIG is business and technical discussion on digital publishing with other media and entertainment topics, e.g., (1) immersive reading of digital publications with immersive web technologies, (2) <a href="https://www.w3.org/2011/audio/">audiobooks</a> in an interoperable manner, and (3) <a href="https://www.w3.org/community/sync-media-pub/">synchronized audio/video</a> with Web Publications.
]]

I think that's nice, and am asking Daihei for confirmation.

Reply:
I asked Daihei about the above proposed text, and he was OK with that.
He just asked us to add a link for "Publishing@W3C" with "https://www.w3.org/publishing/" as well.
If that's OK by you both, I'll add that change and merge this PR :)

Reply:
That's OK for me. Thank you!

Reply:
Thanks, @chrisn !

Reply:
Sure, let's do that!

Reply:
Thanks, @tidoust !

I've just applied the proposed text with the link for "Publishing@W3C", and will merge this PR now :)